common/src/test.rs

```diff
@@ -61,7 +61,7 @@ pub fn logger() {
                     })
                     .with_filter(filter())
             }))
-            .init();
+            .try_init();
```

Comment:
If this is for tests, let's unwrap this result.

Reply:
sometimes we have multiple global loggers initialized, so rather than failing the test it just ignores the result if there's already a logger present. This was an issue in `bindings_ffi`, where the logger is attached in `create_client`